Lesson 26_HW01/lesson 26 script1.js

```diff
@@ -0,0 +1,47 @@
+const appid = '5d066958a60d315387d9492393935c19'
+const container = document.querySelector('.container')
+const btn = document.querySelector('.btn')
+const city = document.querySelector('#city') 
+const units = document.querySelector('#units')
+const weather = document.querySelector('.weather')
+weather.style.display = "none";
+
+btn.addEventListener('click', getData)
+
+
+function getData(e) {
+  const url = `http://api.openweathermap.org/data/2.5/weather?q=${city.value}&units=${units.value}&APPID=${appid}`;
+  e.preventDefault();
+
+  fetch(url)
+    .then((resolve) => {
```

Comment:
```suggestion
    .then((response) => {
```
так буде зрозуміліше